runtime/src/dpe_crypto.rs

```diff
@@ -281,21 +281,31 @@ impl<'a> Crypto for DpeCrypto<'a> {
         match algs {
             AlgLen::Bit256 => Err(CryptoError::Size),
             AlgLen::Bit384 => {
+                // derive an EC key pair from the CDI
+                let (_, pub_key) = Self::derive_key_pair(self, algs, cdi, label, info)?;
```

Comment:
nit: instead of pubkey can we call this `hmac_seed`? And add a comment explaining that the output point must be kept secret and why it is safe to do this (the output point is derived from the private key, so as long as that output is kept secret and not released outside of Caliptra, it is safe to use it as key material.

I want it to be clear that it shouldn't be treated as public data.

Reply:
Done